dev/Kozani/KozaniSendToLocal/KozaniSendToLocalTracelogging.h

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "pch.h"
+
+bool __stdcall wilResultLoggingThreadCallback(const wil::FailureInfo& failure) noexcept;
+
+GUID& GetLifetimeActivityId() noexcept;
+
+class WindowsAppRuntimeKozaniSendToLocal_TraceLogger final : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(
+        WindowsAppRuntimeKozaniSendToLocal_TraceLogger,
+        "Microsoft.WindowsAppRuntime.KozaniSendToLocal",
+        // {75f3d43c-eca9-420b-8600-320ad7e8cc97}
```

Comment:
>{75f3d43c-eca9-420b-8600-320ad7e8cc97} [](http://example.com/codeflow?start=11&length=38)

all of your traceloggers seem to use the same guid with different names. is that intended? i didn't think tracelogger supported that.

Reply:
Good catch! No that wasn't intended. Thanks will fix